CallCenterSample/Controls/SpeechToTextControl.xaml.cs

```diff
@@ -284,14 +285,18 @@ private async Task AnalyzeTextAsync()
                 if (!string.IsNullOrEmpty(this.speechRecognitionTextBox.Text))
                 {
                     SentimentResult textAnalysisResult = await TextAnalyticsHelper.GetTextSentimentAsync(this.translatedText, this.detectedLanguage[0]);
```

Comment:
I'm not sure I understand how the scores and the sentiment of the text is used here, could you explain more?